modules/kargoBidAdapter.js

```diff
@@ -53,6 +55,20 @@ export const spec = {
     }
     return bidResponses;
   },
+  getUserSyncs: function(syncOptions) {
+    const syncs = [];
+    const seed = spec._generateRandomUuid();
+    const clientId = spec._getClientId();
+    if (syncOptions.iframeEnabled && seed && clientId) {
+      for (let i = 0; i < SYNC_COUNT; i++) {
```

Comment:
Just to confirm the intended logic here - this setup will always generate 5 sync requests that are largely the same minus the `idx` param which gets incremented by 1 for each subsequent sync request.  Is that the expected behavior?

Reply:
Yes, we want to attempt 5 syncs (barring a cap set by the configuration) and we want our server to be able to choose what to sync, potentially randomly, using a stateless system that gets passed an instance ID/seed and an incrementer.

Reply:
Thanks for clarifying.